sql/hive/src/main/scala/org/apache/spark/sql/hive/HiveSessionState.scala

```diff
@@ -32,19 +32,9 @@ import org.apache.spark.sql.streaming.StreamingQueryManager
 
 /**
  * A class that holds all session-specific state in a given [[SparkSession]] backed by Hive.
- * @param sparkContext The [[SparkContext]].
- * @param sharedState The shared state.
- * @param conf SQL-specific key-value configurations.
- * @param experimentalMethods The experimental methods.
- * @param functionRegistry Internal catalog for managing functions registered by the user.
  * @param catalog Internal catalog for managing table and database states that uses Hive client for
  *                interacting with the metastore.
- * @param sqlParser Parser that extracts expressions, plans, table identifiers etc. from SQL texts.
  * @param metadataHive The Hive metadata client.
- * @param analyzer Logical query plan analyzer for resolving unresolved attributes and relations.
- * @param streamingQueryManager Interface to start and stop
- *                              [[org.apache.spark.sql.streaming.StreamingQuery]]s.
- * @param queryExecutionCreator Lambda to create a [[QueryExecution]] from a [[LogicalPlan]]
```

Comment:
Each of these is identical to their `SessionState` counterpart and should be inherited by Scaladoc comment inheritance, do we still need them?